docs/release_notes/v1.0.0-rc.1.md

```diff
@@ -0,0 +1,401 @@
+
+### Java SDK
+
+- **FIXED** Save byte[] in actor's state store. [352](https://github.com/dapr/java-sdk/issues/352)
+
+- **FIXED** register timer method to return timer's name, enabling use of auto-generated values. [355](https://github.com/dapr/java-sdk/issues/355)
+
+- **FIXED** withObjectSerializer method to return generic type in ActorProxyBuilder. [356](https://github.com/dapr/java-sdk/issues/356)
+
+- **UPDATED** dependencies expect opentelemetry. [361](https://github.com/dapr/java-sdk/issues/361)
+
+- **FIXED** Timer invocation after Actor runtime app restarts. [362](https://github.com/dapr/java-sdk/issues/362)
+
+- **ADDED** Added codecov rules to ensure that no PR reduces unit test coverage. [364](https://github.com/dapr/java-sdk/issues/364)
+
+- **FIXED** high memory consumption issue in Java SDK for HTTP communication. [370](https://github.com/dapr/java-sdk/issues/370)
+
+- **UPDATED** opentelemetry library to 0.7.1. **CHANGED** repo ID of spring repo to spring. [382](https://github.com/dapr/java-sdk/issues/382)
+
+### Python SDK
+
+- **ADDED** W3C trace headers to Dapr API calls. [97](https://github.com/dapr/python-sdk/issues/97)
+
+- **Updated** dapr python sdk pkg documents. [125](https://github.com/dapr/python-sdk/issues/125)
+
+- **ADDED** missing etag to StateResponse [128](https://github.com/dapr/python-sdk/issues/128)
+
+- **ADDED** Added codecov reports to python-sdk PRs [133](https://github.com/dapr/python-sdk/issues/133)
+
+### Documentation
+
+- **UPDATED** environment-setup.md. [841](https://github.com/dapr/docs/pull/841)
+
+- **ADDED** Performance section in docs [863](https://github.com/dapr/docs/issues/863)
+
+- **FIXED** documentation link to "Sending Messages to Output Bindings" [867](https://github.com/dapr/docs/pull/867)
+
+- **ADDED** Documentation for enabling Dapr to talk to an app via an SSL connection [868](https://github.com/dapr/docs/issues/868)
+
+- **FIXED** dead link to Ignite video session. [878](https://github.com/dapr/docs/pull/878)
+
+- **FIXED** links in page developing-applications/ides/vscode-remotecontainers/. [887](https://github.com/dapr/docs/issues/887)
+
+## Upgrading to Dapr 1.0.0-rc.1
+
+To upgrade to this release of Dapr, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have installed. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if docker command needs sudo:
+
+```bash
+dapr uninstall --all
+```
+
+For RC releases like this, download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have installed the CLI, run:
+
+```bash
+dapr init
```

Comment:
As per our discussion this will be **dapr init --version=1.0.0-rc.1** otherwise even the 1.0.0-rc.1 will pull down the 0.11.3 release